### PR TITLE
Turkish support

### PR DESCRIPTION
**Pull request by DerinHalil**
Also fixed some stuff in English (sorry, Moonded, I know you hate grammar)

### CODE REVIEW

File: nuxt.config.js [hunk 1]
```diff
@@ -111,6 +111,11 @@ export default {
         file: 'ja-JP.js',
         name: '日本語'
       }
+      {
+        code: 'tr'
+        file: 'tr-TR.js'
```
Comment (zSoulweaver) [inline nuxt.config.js]: Commas after the value, as well as before new object.

```js
},
{
  code: 'tr',
  file: 'tr-TR.js',
  ...
```

**Comment by zSoulweaver** [comment]
The blog post thumbnail images are now also mismatched between the `2020-12-17-cyberpunk-update-no1` and `2020-12-14-cyberpunk-modding-status` directory.

**Comment by DerinHalil** [comment]
> The blog post thumbnail images are now also mismatched between the `2020-12-17-cyberpunk-update-no1` and `2020-12-14-cyberpunk-modding-status` directory.

blogpost1.jpg was attached to the second post and blogpost2.jpg to the first. Was that correct? If so, why were the images named like that?

**Comment by zSoulweaver** [comment]
> > The blog post thumbnail images are now also mismatched between the `2020-12-17-cyberpunk-update-no1` and `2020-12-14-cyberpunk-modding-status` directory.
> 
> blogpost1.jpg was attached to the second post and blogpost2.jpg to the first. Was that correct? If so, why were the images named like that?

I wouldn't know sorry, I didn't create the blog posts, that would be @Moonded. Although the page titles no longer match what is in the picture, so I imagine that they were just named incorrectly but in the right directories.

If you could name the files correctly, but put them back in the right directory, and fix the syntax issue in the config file I'll merge this 👍

**Comment by DerinHalil** [comment]
Hm yeah, I don't think I'm allowed to make that change in the same PR though, I'd have to make another one. Maybe you have the permission to do that?

**Comment by zSoulweaver** [comment]
@DerinHalil You just need to push the changes to your branch and GitHub will pick up them up

File: content/blog/2020-12-17-cyberpunk-update-no1/locales/en.md [hunk 2]
```diff
@@ -1,32 +1,33 @@
...
+description: It's been a while, and with time comes change! The first updates regarding modding.
 headerImage:
-thumbnailImage: blogpost1.jpg
+thumbnailImage: blogpost2.jpg
 headerColor: 10, 70, 230
 category: Cyberpunk General Modding
 createdAt: 2020-12-17
 ---
 
-Hey Choombas!
```
Comment (Moonded) [inline content/blog/2020-12-17-cyberpunk-update-no1/locales/en.md:11]: "Hey Choombas!" was correct, it's not "Hey, choombas!".

**Comment by DerinHalil** [comment]
Okay, I think I've done that now.

**Review by Moonded — APPROVED** [review]

**Comment by zSoulweaver** [comment]
Files look good, just need to fix up the config file now and it's all good.

**Comment by DerinHalil** [comment]
Should be done.

**Comment by zSoulweaver** [comment]
> Should be done.

Not quite, on mobile so I can't give you a good example, but have a look at the previous items for an example. You'll need a comma after the code value and a comma after the previous curly bracket.

When I get home I can just fix it and commit it to the PR for you if you want.

**Pull request merged**